package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "chesster",
+  "version": "1.0.0",
```

Comment:
we are officially 1.0? woohoo! we made it out of alpha and beta without even knowing it!